src/chat/__tests__/chatReducers-test.js

```diff
@@ -656,5 +656,25 @@ describe('chatReducers', () => {
 
       expect(newState).toEqual(expectedState);
     });
+
+    test('when replaceExisting is true, common messages are not replaced', () => {
```

Comment:
It is possible that the message has been changed, even though the id is the same the message contents will not be.
I guess we need to handle that.